packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -220,10 +224,13 @@ public void onResult(@NonNull GoogleSignInResult signInResult) {
      * were requested.
      */
     public void signIn(Result result) {
+      if (mRegistrar.activity() == null) {
```

Comment:
When would this happen? If it makes no sense to call `signIn` when there is no foreground activity, maybe we should throw an `IllegalStateException` instead?

Reply:
Done.